src/types.ts

```diff
@@ -89,7 +89,7 @@ export interface Action {
     /** If action won't wait for an user interaction to continue to the next one. */
     autoskip?: boolean;
     /** Action's execution function. */
-    exec: () => void | Promise<void>;
+    exec: (next:Function) => void | Promise<void> | Action;
```

Comment:
This is changed in the next PR. See [this message](https://discord.com/channels/1051990700788555806/1052055170915254333/1081086716884037732) in the TCFS Discord